build.sbt

```diff
@@ -208,7 +158,7 @@ lazy val `benchmarks-vprev` = project
 
 lazy val `benchmarks-vnext` = project
   .in(file("benchmarks/vnext"))
-  .dependsOn(channel)
+  .dependsOn(`rpc-service`)
```

Comment:
You can delete this one because it's a transitive dependency of `server`

Reply:
👍 https://github.com/higherkindness/mu-scala/pull/900/commits/585bbc2172172de2b4ea77dc14580fbc4b8e2d4d